### PR TITLE
documentation: Add example for various popular public registries

### PR DESCRIPTION
**Pull request by Poulpatine**
**Which issue does this PR fix**:
No issue, only documentation to give examples for most popular public registries.

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:
At least one image has been pulled from each registry.

**Automation added to e2e**:
N/A

**Will this break upgrades or downgrades?**
No

**Does this PR introduce any user-facing change?**:
No

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/config-popular-registries.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  },
+  "http": {
+    "address": "0.0.0.0",
+    "port": "5000"
+  },
+  "log": {
+    "level": "info"
+  },
+  "sync": {
+    "credentialsFile": "/config/credentials.json",
```
Comment (peusebiu) [inline examples/config-popular-registries.json:14]: no need for this one, all registries are public

File: examples/config-popular-registries.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{
+  "distSpecVersion": "1.0.1",
```
Comment (peusebiu) [inline examples/config-popular-registries.json:2]: 1.1.0-dev

File: examples/config-popular-registries.json (hunk 1 above)
Comment (peusebiu) [reply examples/config-popular-registries.json:14]: nevermind, they can be used to not hit rate limit.

**Comment by peusebiu** [comment]
can you also update this one: https://github.com/project-zot/zot/blob/main/test/blackbox/sync_docker.bats ?

There's a prefix filtering on all registries because we want to make sure that all registries are hit/used in the tests while syncing.

**Comment by andaaron** [comment]
With regards to the commit message, we have some checks:
1. Please start the first line with 'docs', not 'doc'. I may look like the same, but the job is configured to use https://www.conventionalcommits.org/en/v1.0.0/
2. Please use `git commit -s -S` for signing

**Comment by Poulpatine** [comment]
> can you also update this one please: https://github.com/project-zot/zot/blob/main/test/blackbox/sync_docker.bats ? if not, I'll do it after merging it.
> 
> There's a prefix filtering on all registries because we want to make sure that all registries are hit/used in the tests while syncing.

I'd do that gladly but I'm not sure to understand what you want ?

File: examples/config-popular-registries.json (hunk 2 above)
Comment (Poulpatine) [reply examples/config-popular-registries.json]: Fixed.

File: examples/config-popular-registries.json (hunk 1 above)
Comment (Poulpatine) [reply examples/config-popular-registries.json:14]: Hence, done.

**Comment by andaaron** [comment]
@Poulpatine `git commit --amend -s -S`, both the `Signed-off-by` and signing with the GPG Key.

**Comment by Poulpatine** [comment]
> @Poulpatine `git commit --amend -s -S`, both the `Signed-off-by` and signing with the GPG Key.

Sure, let me fix that mismatch between my GPG key and my commit email and.

**Comment by rchincha** [comment]
{"level":"error","config":"examples/config-popular-registries.json","time":"2023-06-23T16:03:48Z","message":"Config file is invalid"}
panic: config: invalid config

^ `zot` has a "verify" subcmd which can be used to validate the config file like so.
`zot verify config-file` 

Also, your commits need to be signed with a "Signoff" in commit message.

**Comment by andaaron** [comment]
hi @Poulpatine, please rebase locally. The merge commit messages are not compliant with the standard I mentioned above, so the check is failing.

**Comment by Poulpatine** [comment]
> hi @Poulpatine, please rebase locally. The merge commit messages are not compliant with the standard I mentioned above, so the check is failing.

Hi @andaaron ,

Ok, done.

Thanks.

**Comment by andaaron** [comment]
> > hi @Poulpatine, please rebase locally. The merge commit messages are not compliant with the standard I mentioned above, so the check is failing.
> 
> Hi @andaaron ,
> 
> Ok, done.
> 
> Thanks.

Sure, no problem. Please also upload your new public signing key to your GH profile, so the commit shows as verified.

**Comment by andaaron** [comment]
@Poulpatine thank you for your contribution, again!

**Pull request merged**